frontend/packages/client/src/contexts/NotificationService.js

```diff
@@ -1,13 +1,31 @@
 import { createContext, useContext, useEffect, useState } from 'react';
+import { subscribeNotificationIntentions } from 'const';
 import { useWebContext } from './Web3';
 
 const NotificationServiceContext = createContext({});
 
 const INIT_NOTIFICATION_SETTINGS = {
   walletId: '',
   email: '',
-  communitySubscription: [],
-  isUnsubscribedFromCommunityUpdates: false,
+  communitySubscription: [{ communityId: '1', subscribed: true }],
```

Comment:
This is for rendering the community list on settings page, demo purpose only, will remove once api is implemented